packages/accounts-base/accounts_client_tests.js

```diff
@@ -94,6 +94,20 @@ Tinytest.addAsync(
   }
 );
 
+Tinytest.addAsync(
+  'accounts async - Meteor.loggingIn() is false after login has completed',
+  (test, done) => {
+    logoutAndCreateUser(test, done, () => {
+      // Login then verify loggingIn is false after login has completed
+      Meteor.loginWithPassword(username, password, async () => {
+        test.isTrue(await Meteor.userAsync());
+        test.isFalse(Meteor.loggingIn());
```

Comment:
I'd rather swap these two, as if the `userAsync` may cause an additional wait, `loggingIn` can "flicker" but this test will still pass.